tv/pages/speakers/[id].tsx

```diff
@@ -0,0 +1,48 @@
+import { getSessions } from '../../api/queries/getSessions';
+import { getSpeakers, getSpeakerById } from '../../api/queries/getSpeakers';
+import { Speaker } from '../../interfaces/speaker';
+import { Session } from '../../interfaces/session';
+import SpeakerDisplay from '../../components/SpeakerDisplay';
+import { Params } from '../../interfaces';
+
+type SpeakersProps = {
+  sessions: Session[];
+  speaker: Speaker;
+};
+
+export declare type SpeakersParams = {
+  [param: string]: Params;
+};
+
+export default function Speakers(props: SpeakersProps) {
```

Comment:
As this is an [id] page to display an individual item, the function and types names should be singular. Speaker, SpeakerParams, SpeakerProps.

Reply:
Fixed